<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1700</width>
    <height>1300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1400</width>
    <height>1300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="SearchButton">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>230</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>查询路线</string>
    </property>
   </widget>
   <widget class="QLabel" name="Golabel">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>70</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;出发点：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Terminallabel">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>150</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;目的地：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="BikeTimelabel">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>330</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;预计骑行时间：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="BikeTimeEdit">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>330</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="WalkTimelabel">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>400</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;预计步行时间：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="WalkTimeEdit">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>400</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="GocomboBox">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>70</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>无</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>东门</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>西门</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>南门</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>博雅塔</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>图书馆</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>百周年纪念讲堂</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>新太阳学生中心（文创店）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>静园（革命烈士纪念碑）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>勺园</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>农园</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>家园</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="TerminalcomboBox">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>140</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>无</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>东门</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>西门</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>南门</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>博雅塔</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>图书馆</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>百周年纪念讲堂</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>新太阳学生中心（文创店）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>静园（革命烈士纪念碑）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>勺园</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>农园</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>家园</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="Shortestlabel">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>490</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;寻找最近的：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ShortestcomboBox">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>490</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>无</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>食堂</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>大门</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="ShortestButton">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>570</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>确认查找</string>
    </property>
   </widget>
   <widget class="QLabel" name="Viewlabel">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>740</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;查看近景：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ViewcomboBox">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>740</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>无</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>未名湖</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>博雅塔</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>西门</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>华表</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>图书馆</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>百周年纪念讲堂</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>新太阳学生中心</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>静园草坪</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>第一教学楼</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>蔡元培像</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="ViewButton">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>820</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>确认查看</string>
    </property>
   </widget>
   <widget class="QLabel" name="Welcomelabel">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>950</y>
      <width>521</width>
      <height>401</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#aa0000;&quot;&gt;欢迎来到北京大学！&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#aa0000;&quot;&gt;Welcome to PekingUniversity!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="FindResultlabel">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>640</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;查找结果：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FindResultEdit">
    <property name="geometry">
     <rect>
      <x>1450</x>
      <y>640</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Travellabel">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>930</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;游览路线选择：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="TravelcomboBox">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>930</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>无</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>精华路线（1h）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>经典路线（2h）</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="TravelButton">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>1000</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>确认选择</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1700</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
